deploy/helm/values.yaml

```diff
@@ -274,6 +274,12 @@ trivy:
   #
   # serverCustomHeaders: "foo=bar"
 
+  # existingSecret if a secret containing gitHubToken, serverToken or serverCustomHeaders has been created outside the chart (e.g external-secrets, sops, etc...).
```

Comment:
since it a helm setting maybe its the best to put it under [operator](https://github.com/aquasecurity/trivy-operator/blob/main/deploy/helm/values.yaml#L25)